libraries/cyclestreets-core/build.gradle

```diff
@@ -9,8 +9,6 @@ dependencies {
   api 'org.osmdroid:osmdroid-android:6.1.6'
   api 'org.slf4j:slf4j-android:1.7.30'
 
-  api "com.android.support:support-compat:${rootProject.ext.supportLibraryVersion}"
```

Comment:
the definition of `supportLibraryVersion` can probably be removed from the root `build.gradle`